code/modules/mining/ores_coins.dm

```diff
@@ -369,6 +369,8 @@
 	return w_type
 
 /obj/item/weapon/coin/is_screwdriver(var/mob/user)
```

Comment:
this is use in more than just screwing operations, adding a fluff line to an is_check is a big nono imo

Reply:
what do you suggest instead

Reply:
Personally I'd have `[tool]_act()` functions or something and put it in there, but that would be a significant refactor

Reply:
@DamianX tbh just anything else, screwdriver_act would make sense to do